src/worker/browser-env/runner/test-runner/constants.ts

```diff
@@ -1 +1,11 @@
 export const WORKER_EVENT_PREFIX = "worker";
+
+export const SUPPORTED_ASYMMETRIC_MATCHER = {
```

Comment:
Why do you use `enum` in `browser-env/vite/types` and in `browser-env/vite/browser-modules/types.ts`, but `export const ... = {...} as const` here?

Reply:
Just used it from wdio source code. To my mind there doesn't seem to be much difference between `enum` and `as const`